websocket/polygon.go

```diff
@@ -0,0 +1,63 @@
+package polygonws
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/polygon-io/client-go/websocket/client"
+	"github.com/polygon-io/client-go/websocket/models"
+)
+
+// PolygonClient defines a client to the Polygon WebSocket API.
+type PolygonClient struct {
+	client.Client
+}
+
+func New(apiKey string) *PolygonClient {
+	c := client.New(apiKey)
+	return &PolygonClient{
+		Client: c,
+	}
+}
+
+// StreamSecondAggs streams real-time second aggregates for a given ticker symbol. "*" will stream data for
+// all tickers of a given market type. For more details see https://polygon.io/docs/stocks/ws_stocks_a.
+// Callers of this method must close the connection:
+//   conn, aggs, err := c.StreamSecondAggs(context.TODO(), models.Stocks, "AAPL")
+//   if err != nil {
+//       return err
+//   }
+//   defer conn.Close()
+func (pc *PolygonClient) StreamSecondAggs(ctx context.Context, market models.MarketType, ticker string) (*client.Conn, models.SecondAggs, error) {
```

Comment:
i think we need to create and return a new connection here because a connection should only handle one type of data and we need the user to call `defer conn.Close()` on it. this method handles creating a channel buffer for the user (i could also see arguments for letting the user do that and pass it in). it also assumes that users will stop the connection/loop with a context. definitely looking for feedback on this part!